cmd/run.go

```diff
@@ -117,10 +117,20 @@ func (c *cmdRun) run(cmd *cobra.Command, args []string) error {
 	// Spin up the REST API server, if not disabled.
 	if c.gs.flags.address != "" {
 		initBar.Modify(pb.WithConstProgress(0, "Init API server"))
+
+		apiWG := &sync.WaitGroup{}
+		apiWG.Add(2)
+		defer apiWG.Wait()
+
+		srvCtx, srvCancel := context.WithCancel(globalCtx)
+		defer srvCancel()
+
+		// TODO: send the ExecutionState and MetricsEngine instead of the Engine
```

Comment:
I see this is not removed from the latest PR, can we open an issue for it?

Reply:
Ah, I forgot to delete the comment in https://github.com/grafana/k6/pull/2813, added this as a reminder: https://github.com/grafana/k6/pull/2813/files#r1065929155